### DISCUSSION TITLE
change in expand arrow of Disclosure component

### DISCUSSION THREAD

**Discussion by jeevan-sg** [twilio-labs/paste#1561] [General] [answered]
Disclosure component:  when same sample code is copied from [docs](https://paste.twilio.design/components/disclosure), the component renders with a different 'expand arrow' . Anything am missing here?
<img width="290" alt="Screen Shot 2021-06-11 at 3 16 02 PM" src="https://user-images.githubusercontent.com/45638853/121754588-dbcfd480-cac9-11eb-8711-5a9987af7b7b.png">
<img width="257" alt="Screen Shot 2021-06-11 at 3 26 09 PM" src="https://user-images.githubusercontent.com/45638853/121754593-dc686b00-cac9-11eb-867b-86bc7917658e.png">

**Comment by richbachman** [comment]
Hi @jeevan-sg. Thanks for posting this here. Based off the screenshots, it looks like you might be using an older version of the Disclosure. Would you mind verifying the version you're using? Thanks.

**Reply by jeevan-sg** [reply]
Hi @richbachman, the version used in our app is: "@twilio-paste/core": "4.3.1" - it seems bit old. can you advise what is the latest version? thanks

**Comment by richbachman** [comment, marked as answer]
Yeah. That version is a bit behind. Currently core is on 9.6.0. You can find the package version here: https://paste.twilio.design/getting-started/engineering#paste-packages

**Reply by jeevan-sg** [reply]
Thank you. 

does upgrading core package requires upgrading other packages as well? because I am running into below errors: 

```
npm ERR! While resolving: twilio-console@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.6" from @twilio-paste/styling-library@0.3.1
npm ERR! node_modules/@twilio-paste/styling-library
npm ERR!   @twilio-paste/styling-library@"^0.3.0" from the root project
npm ERR!   peer @twilio-paste/styling-library@"^0.3.1" from @twilio-paste/icons@5.3.0
npm ERR!   node_modules/@twilio-paste/icons
npm ERR!     @twilio-paste/icons@"5.3.0" from the root project
npm ERR!     1 more (@twilio-paste/core)
npm ERR!   2 more (@twilio-paste/box, @twilio-paste/style-props)
```

**Reply by richbachman** [reply]
Can you provide your package.json, so we can help troubleshoot?

**Reply by jeevan-sg** [reply]
sure, 
https://github.com/twilio/console/blob/main/package.json

**Reply by richbachman** [reply]
Hi @jeevan-sg. Not 100% sure what the issue is, but we're working on a PR for the console repo to get Paste fully upgraded. We'll report back soon.

**Reply by richbachman** [reply]
Hi @jeevan-sg. My teammate @TheSisb has a PR in to upgrade Paste: https://github.com/twilio/console/pull/134

**Reply by jeevan-sg** [reply]
thanks @richbachman for quick turnaround. Appreciate that.
